volumes/vds/sql/functions/create-function-aggregate_15min_vds_volumes_bylane.sql

```diff
@@ -0,0 +1,63 @@
+CREATE OR REPLACE FUNCTION vds.aggregate_15min_vds_volumes_bylane(_start_date timestamp, _end_date timestamp)
+    RETURNS void
+    LANGUAGE 'plpgsql'
+
+    COST 100
+    VOLATILE SECURITY DEFINER 
+AS $BODY$
+
+BEGIN
+	
+    --Aggregated into speed bins and 1 hour bin
+    INSERT INTO vds.volumes_15min_bylane (division_id, vds_id, detector_id, lane, datetime_bin, volume_15min, expected_bins, num_obs)
+    
+    WITH detector_inventory AS (
+        SELECT *, 
+            CASE WHEN detector_id LIKE 'D%' AND division_id = 2
+                THEN 45 --20 sec bins
+            WHEN detector_id LIKE ANY ('{"YONGE HEATH%", "YONGE DAVISVILLE%", "BCT%"}')
+                THEN 1 --15 min bins
+            WHEN detector_id LIKE ANY ('{"YONGE & DAVENPORT SMARTMICRO%", "%YONGE AND ROXBOROUGH%"}')
+                THEN 3 --5 min bins
+            END AS expected_bins
+        FROM vds.vdsconfig 
+    )
+
+    SELECT 
+        d.division_id,
+        d.vds_id,
+        c.detector_id,
+        d.lane,
+        d.datetime_15min,
+        SUM(d.volume_veh_per_hr) / 4 / c.expected_bins AS volume_15min,
+            -- / 4 to convert hourly volume to 15 minute volume
+            -- / (expected_bins) to get average 15 minute volume depending on bin size (assumes blanks are 0)
+        c.expected_bins,
+        COUNT(*) AS num_obs
```

Comment:
so we can get an indication of the quality of these averages based on how much expected_bins and num_obs are close, right? Don't remember if I saw this in the README. should we highlight that some of these 15-min counts might not be so accurate?